styles.css

```diff
@@ -1,3 +1,3 @@
 .header {
-  background-color: red;
+  background-color: blue;
```

Comment:
- [ ] As said in the project requirement, you should give the text a color, not a background color.
`Also make sure your CSS and HTML  files are connected correctly`